test/e2e/gardener/shoot/internal/shoot.go

```diff
@@ -81,20 +81,29 @@ func ItShouldCreateShoot(s *ShootContext) {
 	}, SpecTimeout(time.Minute))
 }
 
+// ItShouldReadShootFromAPIServer reads the Shoot from the API server.
+func ItShouldReadShootFromAPIServer(s *ShootContext) {
```

Comment:
According to my previous comments, we should probably remove this shared `It` in favor of reading in the `It`s where we perform the actual assertions.